Assets/HoloToolkit/Input/Scripts/SelectedObjectMessageSender.cs

```diff
@@ -0,0 +1,46 @@
+﻿using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// SelectedObjectMessageSender class sends a Unity message to currently selected object.
+/// Object selection is controlled via OnSelectObject and OnClearSelection events.
+/// Object selection and messages can be triggered using keyword responses registered in KeywordManager.
+/// </summary>
+public class SelectedObjectMessageSender : MonoBehaviour
```

Comment:
Ultimately I like that any number of things can trigger this. Or are you asking how this will be configured in the test scene?